continuous-testing-foundations/dotnet/Pragmatest.Wallets/src/Pragmatest.Wallets.Web/Controllers/WalletController.cs

```diff
@@ -34,10 +34,15 @@ public async Task<ActionResult<BalanceResponse>> Balance()
         }
 
         [HttpPost]
-        public async Task<IActionResult> Deposit(DepositRequest depositRequest)
+        public async Task<ActionResult<BalanceResponse>> Deposit(WithdrawalRequest depositRequest)
```

Comment:
Why is the deposit endpoint taking a WithdrawalRequest?

Reply:
Fixed